crates/nu-test-support/src/fs.rs

```diff
@@ -244,27 +88,28 @@ pub fn binaries() -> PathBuf {
     };
 
     std::env::var("CARGO_TARGET_DIR")
-        .map(PathBuf::from)
-        .unwrap_or_else(|_| root().join("target"))
+        .ok()
+        .and_then(|p| AbsolutePathBuf::try_from(p).ok())
+        .unwrap_or_else(|| root().join("target"))
         .join(build_target)
         .join(profile)
 }
 
-pub fn fixtures() -> PathBuf {
-    root().join("tests").join("fixtures")
-}
-
-pub fn assets() -> PathBuf {
-    root().join("tests/assets")
+pub fn fixtures() -> AbsolutePathBuf {
+    let mut path = root();
+    path.push("tests");
+    path.push("fixtures");
+    path
 }
 
-pub fn in_directory(str: impl AsRef<Path>) -> String {
-    let path = str.as_ref();
-    let path = if path.is_relative() {
-        root().join(path)
-    } else {
-        path.to_path_buf()
-    };
+// FIXME: re-enable nu_json tests
```

Comment:
what does this mean? are you saving this fix for another pr or does it not work or what exactly?

Reply:
`assets` was only used for `nu_json` tests. Someone commented out all the `nu_json` tests, but forgot about this function. I just copied the fixme comment to over here.